public/app/plugins/datasource/tempo/configuration/TagsTimeRangeSettings.tsx

```diff
@@ -0,0 +1,45 @@
+import { DataSourcePluginOptionsEditorProps, updateDatasourcePluginJsonDataOption } from '@grafana/data';
+import { Combobox, InlineField, InlineFieldRow, useStyles2 } from '@grafana/ui';
+
+import { TempoJsonData } from '../types';
+
+import { getStyles } from './QuerySettings';
+
+interface Props extends DataSourcePluginOptionsEditorProps<TempoJsonData> {}
+
+export const DEFAULT_TIME_RANGE_FOR_TAGS = 1800; // 60 * 30
+
+export function TagsTimeRangeSettings({ options, onOptionsChange }: Props) {
+  const styles = useStyles2(getStyles);
+
+  const selectOptions = [
+    { label: 'Last 30 mins of selected range', value: DEFAULT_TIME_RANGE_FOR_TAGS },
```

Comment:
Nit: 

```suggestion
    { label: 'Last 30 minutes of selected range', value: DEFAULT_TIME_RANGE_FOR_TAGS },
```

I think we mostly use "minutes" in the UI instead of abbreviations.